Makefile

```diff
@@ -413,14 +460,17 @@ test-e2e-local-migration: $(GINKGO)
 test-e2e-local: $(GINKGO)
 	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter="default"
 
+test-e2e-local-ha: $(GINKGO)
+	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter "simple || (high-availability && upgrade-to-$(SHOOT_FAILURE_TOLERANCE_TYPE))"
```

Comment:
Can we introduce two `make` targets here as well? We have done it for all other targets, so let's do it also here. Then we you don't need to use the `SHOOT_FAILURE_TOLERANCE_TYPE` variable here anymore.